stable/artifactory/values.yaml

```diff
@@ -37,6 +39,13 @@ database:
     ##   GKE, AWS & OpenStack)
     ##
     # storageClass: "-"
+  resources:
```

Comment:
We recommend not to specify defaults for resources. This should be a conscious choice of the user. Plus, it makes it easier to test charts on small systems, such as Minikube. Do it like so:

```yaml
resources: {}
  # requests:
    # memory: "512Mi"
    # cpu: "100m"
  # limits:
    # memory: "1Gi"
    # cpu: "500m"
```